client/analysis/output.go

```diff
@@ -44,6 +44,7 @@ func prepareSecurityTestResult(container types.Container) {
 	case "npmaudit":
 		prepareNpmAuditOutput(container.COutput, container.CInfo)
 		outputJSON.JavaScriptResults.NpmAuditResult = javaScriptResults.NpmAuditResult
+	case "gitauthors":
```

Comment:
Is this case supposed to be empty? 🧐

Reply:
Good catch, @Takehime! I have added this line so that huskyCI client won't crash in this switch case. 

I have just created an issue (#314) so that we can do a mega refactoring in huskyCI client as vulnerabilities results will be already in MongoDB.